src/index.js

```diff
@@ -1,5 +1,6 @@
 import './style.css';
 
+// Generate HTML
 function component() {
   const element = document.createElement('section');
```

Comment:
- [ ] Good job working on this, but please kindly ensure that you use ES6 arrow functions.

Reply:
@AnselemOdims Oh! I missed that one! thank you for noticing hahaha, I'll correct it right away 
![image](https://user-images.githubusercontent.com/92229666/232248042-3e79a1b7-b512-43a5-8a07-88456bf4eb19.png)